extension/src/paymentHandler/payment-handler.js

```diff
@@ -39,9 +41,31 @@ function _getPaymentHandlers (paymentObject) {
     handlers.push(getPaymentMethodsHandler)
   if (paymentObject.custom.fields.makePaymentRequest && !paymentObject.custom.fields.makePaymentResponse)
     handlers.push(makePaymentHandler)
+  if (paymentObject.custom.fields.makePaymentResponse
```

Comment:
would you consider mobving other checks also in the method, it's not easy to reason 3 checks in one time, and maybe rename the method with isSubmitAdditionalPaymentDetailsRequrest without new.

Reply:
I don't think this would make it easier if I move these conditions to another method. I think it's quite clear that here you try to check for `submitPaymentDetailsHandler`.